src/editor/commands.tsx

```diff
@@ -137,5 +139,7 @@ export async function shouldSendAnyway(commandText: string): Promise<boolean> {
         button: _t("Send as message"),
     });
     const [sendAnyway] = await finished;
+    // To re-focus the composer after QuestionDialog is closed
+    dis.fire(Action.FocusSendMessageComposer);
```

Comment:
This won't work when executing an unknown command in the thread panel, there it'll focus the wrong composer

Reply:
got it, will cover that case as well. Thanks